README.md

```diff
@@ -29,23 +31,14 @@ The documentation is available at
 
 Community
 ---------
-You can join these groups and chats to discuss and ask Akka related questions:
 
-- Forums: [discuss.akka.io](https://discuss.akka.io)
-- Chat room about *using* Akka HTTP: [![gitter: akka/akka][gitter-user-badge]][gitter-user]
-- Q&A: [![stackoverflow: #akka-http][stackoverflow-badge]][stackoverflow]
-- Issue tracker: [![github: akka/akka-http][github-issues-badge]][github-issues] (Please use the issue
-  tracker for bugs and reasonable feature requests. Please ask usage questions on the other channels.)
+If you have questions about the contribution process or discuss specific issues, please interact with the community using the following resources.
 
-All of our forums, chat rooms, and issue trackers are governed by our [Code Of Conduct](https://www.lightbend.com/conduct).
-
-In addition to that, you may enjoy following:
-
-- The [news](https://akka.io/blog/news-archive.html) section of the page, which is updated whenever a new version is released
-- The [Akka Team Blog](https://akka.io/blog)
-- [@akkateam](https://twitter.com/akkateam) on Twitter
-- Projects built with Akka HTTP: [![Built with Akka HTTP][scaladex-badge]][scaladex-projects]
+- [GitHub discussions](https://github.com/apache/incubator-pekko-http/discussions): for questions and general discussion.
+- [Pekko dev mailing list](https://lists.apache.org/list.html?dev@pekko.apache.org): for Pekko development discussions.
+- [GitHub issues](https://github.com/apache/incubator-pekko-http/issues): for bug reports and feature requests. Please search the existing issues before creating new ones. If you are unsure whether you have found a bug, consider asking in GitHub discussions or the mailing list first.
```

Comment:
Shall we put a mention to the slack channel here as well?

Reply:
This is copied from main pekko repo.

Also, ASF Slack is for ASF committers - it isn't for anyone to just sign up too.

I's prefer to just to include the public means of communication.

Reply:
> I'd prefer to just to include the public means of communication.

That's odd, I am part of the ASF slack channel for Kafka and you don't need to be a committer to be in it (I'm not a Kafka committer).

Regardless I'll just approve the PR, we can figure this out later if possible.